rhoas-cli-kafka/README.adoc

```diff
@@ -0,0 +1,265 @@
+////
```

Comment:
This content is unchanged It is was just separated as separate chapter (kafka specific)

Reply:
I don't see the install and login content here. I thought we were replicating the install and login in both getting strated guides for this release?

Reply:
We have 3 tutorials
CLI install
RHOAS 
RHOSAK

CLI install is basically independent and really lengthy. Putting it on beggining of every service guide is not good idea IMHO

Reply:
Okay